app/plugin_api_tests/manual.test_load_configuration_defaults_plugin/main.go

```diff
@@ -40,7 +40,8 @@ func (p *MyPlugin) MessageWillBePosted(_ *plugin.Context, _ *model.Post) (*model
 	if !p.configuration.MyBoolSetting {
 		return nil, "MyBoolSetting has invalid value"
 	}
-	return nil, "OK"
+
+	return &model.Post{}, "OK"
```

Comment:
I'm not sure I understand why are we doing this. What is the error you are seeing?

Reply:
`is always nil (unparam)`. 

The method is returning `nil` in every if condition and also in final return. So, the linter is giving `is always nil error`.

Reply:
I see the issue now. Since we are not using the post param at all, we don't have an option. I think returning `nil` makes more sense than returning an empty post. Let's add an ignore rule in .golangci.yml for this.

Something like for all file paths under `plugin_api_tests/`, ignore the `is always nil` error.